### PR TITLE
Handle /release requests

### LINKED ISSUE #185: Add a /current route that we can use in various places
At times we need the ability to link to "the most current" version of api docs (see https://github.com/ember-learn/ember-api-docs/issues/184 for an example). We need to add a `/current` route to this Ember app that will display the most current version of the api docs.

### PR DESCRIPTION
**Pull request by MartinMalinda**
should fix #185

- [x] get latest project version when visiting /release
- [x] add unit and/or acceptance test
- [x]  make sure that links on /release also link to /release (for SEO reasons)
- [x] get LTS version when visiting /lts

### CODE REVIEW

**Comment by sivakumar-kailasam** [comment]
@MartinMalinda I'm unsure how but would it be possible to research on how the /cuurent paths show up on search engine results first?

**Comment by MartinMalinda** [comment]
But besides that, if the guides and external sources will mostly link to `current`, then it should appear on top.

Maybe we could also have 301 redirect on the latest version to the `current` url.

**Comment by acorncom** [comment]
> I think I read that there is a plan to have no-follow instruction for robots on all but latest project-version

Yes, I think that's what we're going to need to do. We'll likely just handle that via a `robots.txt` file for now ...

**Comment by sivakumar-kailasam** [comment]
Do bots lookup robots file in subpaths too?

**Comment by acorncom** [comment]
@sivakumar-kailasam Don't believe `robots.txt` files are looked up anywhere but root.  We'll need to handle that here as I've currently done: https://github.com/emberjs/website/blob/master/source/robots.txt

**Comment by acorncom** [comment]
@MartinMalinda I imagine you're just starting to recover from EmberConf like the rest of us 😀 But thought I'd poke my head in here to see if we need to do anything else here?

Was talking to Robert Jackson and seems like it might be wiser to shift to a `/release` link instead of a `/current` link here (as current could mean either the latest release or the latest LTS release).  In the long run, we may also want to have a `/lts` link too ... Thoughts?

**Comment by MartinMalinda** [comment]
Yeah, I got excited by Glimmer and invested my "evening/weekend" time into it lately :). I'll try to finish this PR in upcoming days though. Moving from `current` to `release` is fine but I am not sure about `lts`. I guess jsonapi-docs would have to generate isLTS property on project-version model..?

**Comment by sivakumar-kailasam** [comment]
@MartinMalinda I can see how we can add it to the model but we can probably just hardcode it in the app as a config value given that it changes only few times  a year.

File: app/helpers/url-param.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+
+const { inject } = Ember;
+
+export default Ember.Helper.extend({
+
+  '-routing': inject.service(),
```
Comment (sivakumar-kailasam) [inline app/helpers/url-param.js]: May be we can use https://github.com/kellyselden/ember-router-service?

File: config/environment.js [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 module.exports = function(environment) {
   var ENV = {
+    ltsVersion: '2.8.3',
```
Comment (sivakumar-kailasam) [inline config/environment.js]: Should we consider having short circuit to a env variable? That'd probably make it easier to restart app without having to elevate he app.

Comment (MartinMalinda) [inline config/environment.js]: I think I could move straight to https://github.com/emberjs/ember.js/pull/14805. Not sure if it's still under a feature flag though.

Comment (MartinMalinda) [inline config/environment.js]: Nevermind, it does not expose params.

**Comment by toddjordan** [comment]
one thing I noticed is if you go to /release it gets redirected, but if you go to /release/<type>/<name>, it keeps the release in the path.  Is redirecting when just going to /release intentional?

**Comment by toddjordan** [comment]
otherwise looks good.  I'm going to merge and we can update later if need be.

**Pull request merged**